plenum/common/constants.py

```diff
@@ -0,0 +1 @@
+SOCKET_BIND_ERROR_ALREADY_IN_USE = 98
```

Comment:
Lets separate constants by their purpose, at least the ones which indicate error codes, so lets have a module called `error_codes` with such constants, later we can have error messages with a map with keys as above constants